src/Symfony/Bundle/WebProfilerBundle/Resources/config/toolbar.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\WebProfilerBundle\EventListener\WebDebugToolbarListener;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+
+        ->set('web_profiler.debug_toolbar', WebDebugToolbarListener::class)
+            ->tag('kernel.event_subscriber')
```

Comment:
please move `tag` after `arg` to be consistent will all the other migrations

Reply:
done